pkg/ffcapi/api.go

```diff
@@ -197,6 +197,7 @@ type TransactionInput struct {
 	TransactionHeaders
 	Method *fftypes.JSONAny   `json:"method"`
 	Params []*fftypes.JSONAny `json:"params"`
+	Errors []*fftypes.JSONAny `json:"errors"`
```

Comment:
Noting that in `ContractDeployPrepareRequest` we provide the full ABI, so those are already covered there 👍